src/pages/lib/graphqlapi/mutate-data/q/platform/[platform].mdx

```diff
@@ -2,3 +2,566 @@ export const meta = {
   title: `Create, update, and delete application data`,
   description: `Learn more about how to create and update data using GraphQL APIs in Amplify`
 };
+
```

Comment:
@renebrandel This is the one guide where I interwove all the platforms with the new inline filter. This is more a working demo on how this can work and for testing. I thought this might work as an authoring and editing test with you and Jim. I used this one b/c it was the most simple but we may want to consider what this would look like with more cross-platform explainer content. We can also roll this back to be like the next two pages for publication and just link to the mobile fragments